pkgs/build-support/fetchgithub/default.nix

```
@@ -11,6 +14,8 @@ lib.makeOverridable (
 , ... # For hash agility
 }@args:
 
+assert (lib.assertMsg (rev != null) "You must provide `fetchFromGitHub with a `rev` or `tag`.");
```

```suggestion
assert (lib.assertMsg (rev != null) "fetchFromGitHub requires either `rev` or `tag`.");
```

IMO Easier to parse when starting with the cause and not some verbose boilerplate.